src/scripts/hubot-jenkins-notifier.coffee

```diff
@@ -91,17 +96,31 @@ class JenkinsNotifier
     return req.body
 
   process: (req,res) ->
+    console.log "jenkins-notifier: Incoming request at #{req.url}"
+
     query = querystring.parse(url.parse(req.url).query)
 
     res.end('')
 
     envelope = {notstrat:"Fs"}
     envelope.user = {}
-    envelope.user.room = envelope.room = query.room if query.room
+    
     envelope.notstrat = query.notstrat if query.notstrat
     envelope.notstrat = 'FS' if query.always_notify #legacy
     envelope.user.type = query.type if query.type
 
+    if query.user && query.room
+      console.log "Cannot use room and user together"
+      return
+
+    if query.user
+      console.log "sending to user #{query.user}"
+      envelope.user.user = query.user
+
+    if query.room
+      console.log "sending to room #{query.room}"
+      envelope.user.room = envelope.room = query.room if query.room
```

Comment:
Its already wrapped in query.room, so the if isn't ncessary

Can you change the console.log so its optional somehow?